include/comingsoon.css

```diff
@@ -0,0 +1,10 @@
+.bgimg {
+background-image: url('http://www.dreamalittlebigger.com/wp-content/uploads/2013/08/gold-hi-res-glitter-freebie-dream-a-little-bigger.jpg?x32341');
+height: 100%;
+background-position: center;
+background-size: cover;
+position: relative;
+color:white;
+font-family:"Courier New", Courier, monospace;
```

Comment:
Including a font this way assumes that the visitors to your web page have this font installed on their computer which they may or may not. If you want to be 100% sure, you can embed fonts using Google fonts. Just go to the Google Fonts website, find a font you like, and follow their instructions on how to include it on your site.